ReportUtilityBills/resources/util.rb

```diff
@@ -452,8 +452,8 @@ def self.apply_min_charges(monthly_fixed_charge, net_monthly_energy_charge, annu
   # @param end_of_year_bill_credit [Double] end of year bill credit, in USD
   # @param net_metering_excess_sellback_type [String] net metering annual excess sellback rate type
   # @param net_metering_user_excess_sellback_rate [Double] user-specified net metering annual excess sellback rate
-  # @param net_elec [Double] net eletricity production for the run period
-  # @return [Double, Double] end of year bill credit, excess sellback
+  # @param net_elec [Double] net electricity production for the run period
```

Comment:
@joseph-robertson Do you not have spell check built into your editor/IDE? Or maybe this one just slipped past.